src/cli/OutputGTFSCommand.ts

```diff
@@ -31,19 +33,19 @@ export class OutputGTFSCommand implements CLICommand {
       throw new Error(`Output path ${this.baseDir} does not exist.`);
     }
 
-    const associationsP = this.repository.getAssociations();
-    const scheduleResultsP = this.repository.getSchedules();
-    const transfersP = this.copy(this.repository.getTransfers(), "transfers.txt");
-    const stopsP = this.copy(this.repository.getStops(), "stops.txt");
-    const agencyP = this.copy(agencies, "agency.txt");
-    const fixedLinksP = this.copy(this.repository.getFixedLinks(), "links.txt");
+    const associationsP:Promise<Association[]> = this.repository.getAssociations();
+    const scheduleResultsP:Promise<ScheduleResults> = this.repository.getSchedules();
+    const transfersP:Promise<void> = this.copy(this.repository.getTransfers(), "transfers.txt");
```

Comment:
I don't think the typing here is correct. I think that the type will probably be `Promise<Transfer[]>` or something similar.

A `Promise<void>` would imply that there is no object returned by the promise, which is valid syntax, but not what we want here.

Reply:
If I change the type to `Promise<Transfer[]>`, the compiler will complain: `Type 'Promise<void>' is not assignable to type 'Promise<Transfer[]>'`. 

I think the reason is the `copy` method calls `streamToPromise` and `streamToPromise` will return `Promise<void>`. I could change the type of `transfersP` from `Promise<void>` to `Promise<any>` but it still doesn't look right..

Reply:
You're right, I didn't remember this code correctly. 
It will return a `void` because it's just saying that it has finished copying the results to the file.

Reply:
So we should just keep this as is.